core/src/main/scala/org/http4s/ResponseCookie.scala

```diff
@@ -113,7 +114,8 @@ final case class ResponseCookie(
     maxAge.foreach(writer.append("; Max-Age=").append(_))
     domain.foreach(writer.append("; Domain=").append(_))
     path.foreach(writer.append("; Path=").append(_))
-    if (secure) writer.append("; Secure")
+    writer.append("; SameSite").append(sameSite)
```

Comment:
Does this render the `=` somehow?

Reply:
No, that was an oversight on my part. Addressed in 6df6614a9245b2fde1b1af1d394202f23506979a.